warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/IngestJob.java

```diff
@@ -787,8 +787,10 @@ protected void configureBulkPartitionerAndOutputFormatter(Job job, AccumuloHelpe
             conf.set("split.work.dir", conf.get("ingest.work.dir.qualified"));
         }
         conf.setInt("splits.num.reduce", this.reduceTasks);
+        // want validation turned off by default
+        boolean validateShardTableMapFiles = conf.getBoolean(ShardedTableMapFile.SHARD_VALIDATION_ENABLED, false);
```

Comment:
It's just a matter of preference, but I think I would pull the boolean out of the conf in ShardedTableMapFile because the conf is already provided.

Reply:
That makes sense, much cleaner